include/cpp2util.h

```diff
@@ -959,6 +959,15 @@ inline auto to_string(auto&& value, std::string_view) -> std::string
 //  TODO: Does this really warrant a new synonym? Perhaps "is void" is enough
 using empty = void;
 
+//  Type is Type
+//
+
+template <typename X, typename C>
+auto is() -> std::false_type { return {}; }
+
+template <typename X, typename C>
+    requires std::same_as<X, C> || std::derived_from<X,C>
```

Comment:
```suggestion
    requires std::derived_from<X,C>
```
`std::same_as<X, C> <= std::derived_from<X, C>` (implication).

Reply:
Sorry, I was wrong.
`std::same_as` can be `true` for non-class types,
whereas `std::derived_from` can only be `true` for classes.